docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss

```diff
@@ -249,29 +249,58 @@
   }
 
   .hb-dark-pattern &,
-  .hb-dark-inversion .views-element-container & {
+  .hb-dark-inversion .views-element-container &,
+  .hb-dark-pattern.hb-local-footer & {
     color: var(--palette--white);
   }
 
-  // Don't apply these rules to blockquotes in testimonials, the structure is not the same and need
-  // a different implementation.
   &:not(.hb-testimonial__quote) {
-    @include hb-quote-wrapper;
+    position: relative;
+    color: var(--palette--primary);
+    font-style: normal;
+    line-height: hb-calculate-rems(35.56px);
+    text-align: left;
+    padding: hb-calculate-rems(40px);
 
-    > p,
-    > div {
-      margin: 0;
+    &::before,
+    &::after {
+      content: ' ';
+      position: absolute;
+      background-repeat: no-repeat;
+    }
+
+    &::before {
+      @include hb-icon-background('quote-mark', 'tertiary');
+      
+      width: hb-calculate-rems(68px);
```

Comment:
In the figma designs, the quote icons are 36x31px, but the implemented version is a bit larger. It seems that the svgs have some whitespace around the quotes and that makes harder to match the exact size, but try to get a bit more closer to the expected size.